test/unit/selector.js

```diff
@@ -295,6 +295,10 @@ test("class", function() {
 
 	div = jQuery("<div><svg width='200' height='250' version='1.1' xmlns='http://www.w3.org/2000/svg'><rect x='10' y='10' width='30' height='30' class='foo'></rect></svg></div>")[0];
 	equal( Sizzle(".foo", div).length, 1, "Class selector against SVG" );
+
+	var svg = document.createElement("svg");
+	svg = jQuery("<svg width='200' height='250' version='1.1' xmlns='http://www.w3.org/2000/svg'><rect x='10' y='10' width='30' height='30' class='foo'></rect></svg>")[0];
```

Comment:
We already have this covered in the above definition of `div`. The new setup part here can be removed, and the selection scoped to `div.firstChild` instead of a new `svg` variable.

Also, please change the text of the two SVG tests to better differentiate them, something like "Class selector against SVG container" and "Class selector directly against SVG".